tests/chain/burrow.toml

```
@@ -7,6 +7,7 @@
   TendermintRoot = ".burrow"
```

This should be `BurrowDir` now

```suggestion
   BurrowDir = ".burrow"
```